homework_10/exceptions/FileMaxSizeException.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class FileMaxSizeException extends RuntimeException {
```

Comment:
If you're creating a new log file, then this exception is redundant as you won't throw it. You can remove it.